### PR TITLE
Macros all instructions

### PR DESCRIPTION
**Pull request by ochaloup**
Follow-up to PR#1 and https://github.com/marinade-finance/marinade-cli/pull/1.

This PR brings all instruction definitions from https://github.com/marinade-finance/liquid-staking-program to SDK.

These are only definitions. The builder calls and CLI integration require more work.

### CODE REVIEW

**Comment by ochaloup** [comment]
Changed based on changes from PR #1 + adding builder methods for add and remove liquidity.

File: libs/marinade-sdk/src/instructions/add_liquidity.rs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+use borsh::{BorshDeserialize, BorshSerialize};
+use solana_program::pubkey::Pubkey;
+
+#[derive(
+    InstructionData, Clone, Copy, Debug, Default, PartialEq, BorshSerialize, BorshDeserialize,
+)]
+#[discriminator([181, 157, 89, 67, 143, 182, 52, 72])]
+pub struct AddLiquidityData {
+    pub lamports: Option<u64>,
```
Comment (aankor) [inline libs/marinade-sdk/src/instructions/add_liquidity.rs]: It is not an option https://github.com/marinade-finance/marinade-anchor/blob/3b646ca387e5c56755d6482b171324a69948f469/programs/marinade-finance/src/lib.rs#L150

**Review by aankor — CHANGES_REQUESTED** [review]

File: libs/marinade-sdk/src/instructions/add_liquidity.rs (hunk 1 above)
Comment (ochaloup) [reply libs/marinade-sdk/src/instructions/add_liquidity.rs]: +1, changed for the `lamports` field would be just `u64`

**Comment by ochaloup** [comment]
@aankor I closed the pr #1 and this PR contains all changes for to be merged now.

**Comment by ochaloup** [comment]
Added few more builder methods for instructions used at client side added (not tested).

**Pull request merged**